statsmodels/stats/proportion.py

```diff
@@ -113,7 +113,11 @@ def _check(x: np.ndarray, name: str) -> np.ndarray:
     elif method == 'binom_test':
         # inverting the binomial test
         def func(qi):
-            return stats.binom_test(count, nobs, p=qi) - alpha
+            try:
+                return stats.binomtest(count, nobs, p=qi).pvalue - alpha
+            except AttributeError:
```

Comment:
would be better to put the try outside of `func`
e.g. 
try: 
    from scipy stats import binormtest
    def func(qi);
        return binomtest(count, nobs, p=qi).pvalue - alpha
except:
...